packages/react-reconciler/src/__tests__/__snapshots__/ReactIncrementalPerf-test.internal.js.snap

```diff
@@ -356,8 +356,8 @@ exports[`ReactDebugFiberPerf warns if an in-progress update is interrupted 1`] =
 
 ⚛ (Committing Changes)
   ⚛ (Committing Snapshot Effects: 0 Total)
-  ⚛ (Committing Host Effects: 1 Total)
-  ⚛ (Calling Lifecycle Methods: 1 Total)
+  ⚛ (Committing Host Effects: 0 Total)
+  ⚛ (Calling Lifecycle Methods: 0 Total)
```

Comment:
I don't know why this wasn't already 0. There are no host effects or lifecycles in the second flush. New snapshot seems correct.